web/package.json

```diff
@@ -9,7 +9,7 @@
     "lint": "next lint",
     "fix": "next lint --fix",
     "eslint-fix": "eslint --fix",
-    "prepare": "cd ../ && node -e \"if (process.env.NODE_ENV !== 'production'){process.exit(1)} \" || husky install ./web/.husky",
+    "prepare": "cp .env.example .env && cd ../ && node -e \"if (process.env.NODE_ENV !== 'production'){process.exit(1)} \" || husky install ./web/.husky",
```

Comment:
This will cause .env.example to overwrite .env after each installation. I need to think again

Reply:
Use a script to solve the problem. When .evn already has the same key value as .env.example, skip the update to avoid being overwritten.

[`ea3f54f` (#4115)](https://github.com/langgenius/dify/pull/4115/commits/ea3f54f28f3f2231a2fb85fcad6c9251b33f3ee4)